README.md

```diff
@@ -23,6 +23,15 @@ Repository for you to raise a Pull Request to **practice** open-source! 🎉
 
 ## Option 1. Complete this process in Github (in your browser)
 
+```mermaid
+flowchart LR
+
+    Fork[Fork the project]-->branch[Create a New Branch]
```

Comment:
Hey @kartikgajjar7,
I liked your idea to create the flowchart 👍 
Thanks for your contribution ❤️ 
Just a small suggestion, I would also like to add a step "Clone" after "Fork" because without cloning the code on your system, you cannot edit & make changes on your system.

Also, please refer to the suggestions by @Pradumnasaraf & before commit, changes also needs to be staged.
So, the flow should be Fork->Clone->Create new branch->Edit->Take the changes to staging area (git add .)-> commit-> Create PR

Reply:
I second to that!


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient

Reply:
hey, @aparna2071 steps are good. In GitHub UI we don't clone :)